gradle/libs.versions.toml

```diff
@@ -37,6 +37,7 @@ delta-standalone = "3.1.0"
 delta-spark = "3.2.0"
 esotericsoftware-kryo = "4.0.3"
 errorprone-annotations = "2.27.0"
+failsafe = "3.3.2"
```

Comment:
This dependency is quite nice in that it's 0 dependency itself, has Apache licensing and I think there's more use cases in Iceberg to leverage it. For example, I think a lot of the complex logic in Tasks can be simplified.

Furthermore, there's some custom retry logic in JDBC connector which we couldn't use tasks for, but now we could use Failsafe. Wonder what others think

Reply:
I like this a lot